### ISSUE TITLE
App crashes in selection mode with press & hold

### ISSUE THREAD

**Issue by myii** [ubports/terminal-app#64]
Crashes confirmed on two devices:

* N5 `devel`: `2019-02-11`.
* OPO `rc`: `2019-W06`.

Not 100% reproducible but do these steps a few times and it will crash:

1. Press & hold until context menu is displayed.
1. Hit `Select`.
1. Make an arbitrary selection area, doesn't have to involve any text.
1. Press & hold on the selection area.
1. Crash occurs upon releasing the selection area.

Here's a GIF showing the crash occurring on the OPO after the second attempt:

![out](https://user-images.githubusercontent.com/10231489/52603571-466f9780-2e5f-11e9-983b-6380f727ec0e.gif)

**Comment by wagafo** [comment]
In a Nexus 5 I cannot reproduce it with your description, but I have a consistent crash with the following sequence:

1. Press & hold until context menu is displayed.
2. Press `Close` in the menu
3. Crash occurs immediately

**Comment by myii** [comment]
@wagafo There's a certain skill to making it crash the way I've shown in the GIF above!  I'm confident I could make it happen on your device too, if you care to pass it over...!

As for your crash, I'm wondering if that's what the `Close` actually means (while that's very strange).  Perhaps it does mean to close the app, rather than the context menu.  But then that really requires clearer wording.

**Comment by myii** [comment]
@wagafo Well, `terminal.finished()` suggests that it is supposed to close the app:

https://github.com/ubports/terminal-app/blob/b5fbcf1e1cde14a7391fd6667f8aaee004d2f351/src/app/qml/AlternateActionPopover.qml#L108-L112

**Comment by wagafo** [comment]
OK, it is clear (I should have looked at the source). I'll keep trying to see if I can reproduce your crash.

**Comment by wagafo** [comment]
I have made it crash, but after pressing and holding the selected area, I have to slightly move my finger. If I just release it doesn't crash in my device.

**Comment by myii** [comment]
@wagafo Yes, more often than not, the "finger wiggle" seemed to be part of the equation.  But I was able to reproduce it without it, so I didn't want to muddy the main report with too much noise.  Thanks for confirming.

**Comment by myii** [comment]
> OK, it is clear (I should have looked at the source). I'll keep trying to see if I can reproduce your crash.

@wagafo But someone using the terminal doesn't have access to the source code when using it.  And that `Close` option really appears to be for the context menu.  I'm sure it could be worded better.
